src/mpid/common/topo/mpidu_topo.c

```diff
@@ -0,0 +1,34 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/* vim: set ft=c.mpich : */
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+#include <mpidimpl.h>
+#include "mpidu_topo.h"
+
+int MPIDU_topo_map_init(void)
```

Comment:
Can we lose the `map` in the namespace? For the minimum, the namespace and the filename should match.

Reply:
`map` provides some additional indication that this is separate from the `mpir` topology. So I would keep it and change the filename if it's ok with you. I should have done this right away but I realized the `map` in the name space was a little more clear only after I had created the files.